### PR TITLE
Microsoft365readme

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/21282

## Description
Added readme.md to microsof365 feed

## Does it break backward compatibility?
   - [x] Yes

## Must have
- [x] Documentation

### CODE REVIEW

**Comment by Itay4** [comment]
`README.md` should be in the pack root directory, and under the Integrations directory

File: Packs/FeedOffice365/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+## Overview
+---
+
+Use the Office365 feed integration to get indicators from the feed.
+This integration was integrated and tested with version xx of Office365 Feed
```
Comment (Itay4) [inline Packs/FeedOffice365/README.md]: please update `xx` to an actual version

Comment (bakatzir) [reply Packs/FeedOffice365/README.md]: it has no version, just a url you access, e.g:
https://endpoints.office.com/endpoints/Worldwide?ServiceAreas=SharePoint&ClientRequestId=<uuid>

**Comment by bakatzir** [comment]
@kirbles19 Let's set a few minutes regarding it, if we want to add more data regarding the different regions and services for example.

**Comment by kirbles19** [comment]
@orhovy Done
@Itay4 I enhanced the description of the feed, please check it.

**Pull request merged**